### ISSUE TITLE
Gridsync Windows binary fails to launch with qt5reactor error

### ISSUE THREAD

**Issue by ultramancool** [gridsync/gridsync#41]
When launching the windows binary from the GUI it opened a console, spat out some messages and then closed immediately. When running from the CLI it produced the following log:

```
C:\Users\ultra\Desktop\Gridsync-win64\Gridsync>gridsync
PyInstaller Bootloader 3.x
LOADER: executable is C:\Users\ultra\Desktop\Gridsync-win64\Gridsync\Gridsync.exe
LOADER: homepath is C:\Users\ultra\Desktop\Gridsync-win64\Gridsync
LOADER: _MEIPASS2 is NULL
LOADER: archivename is C:\Users\ultra\Desktop\Gridsync-win64\Gridsync\Gridsync.exe
LOADER: No need to extract files to run; setting extractionpath to homepath
LOADER: SetDllDirectory(C:\Users\ultra\Desktop\Gridsync-win64\Gridsync)
LOADER: Already in the child - running user's code.
LOADER: Python library: C:\Users\ultra\Desktop\Gridsync-win64\Gridsync\python35.dll
LOADER: Loaded functions from Python library.
LOADER: Manipulating environment (sys.path, sys.prefix)
LOADER: Pre-init sys.path is C:\Users\ultra\Desktop\Gridsync-win64\Gridsync\base_library.zip;C:\Users\ultra\Desktop\Gridsync-win64\Gridsync
LOADER: sys.prefix is C:\Users\ultra\Desktop\Gridsync-win64\Gridsync
LOADER: Setting runtime options
LOADER: Initializing python
LOADER: Overriding Python's sys.path
LOADER: Post-init sys.path is C:\Users\ultra\Desktop\Gridsync-win64\Gridsync\base_library.zip;C:\Users\ultra\Desktop\Gridsync-win64\Gridsync
LOADER: Setting sys.argv
LOADER: setting sys._MEIPASS
LOADER: importing modules from CArchive
LOADER: extracted struct
LOADER: callfunction returned...
LOADER: extracted pyimod01_os_path
LOADER: callfunction returned...
LOADER: extracted pyimod02_archive
LOADER: callfunction returned...
LOADER: extracted pyimod03_importers
LOADER: callfunction returned...
LOADER: Installing PYZ archive with Python modules.
LOADER: PYZ archive: out00-PYZ.pyz
LOADER: Running pyiboot01_bootstrap.py
LOADER: Running pyi_rth_win32comgenpy.py
LOADER: Running pyi_rth_pkgres.py
LOADER: Running pyi_rth_qt5.py
LOADER: Running pyi_rth_qt5plugins.py
LOADER: Running pyi_rth_twisted.py
LOADER: Running cli.py
libpng warning: iCCP: profile 'icc': 0h: PCS illuminant is not D50
libpng warning: iCCP: profile 'icc': 0h: PCS illuminant is not D50
Traceback (most recent call last):
  File "site-packages\qt5reactor.py", line 402, in iterate
  File "site-packages\qt5reactor.py", line 308, in doIteration
  File "site-packages\qt5reactor.py", line 393, in timeout
TypeError: unorderable types: float() < NoneType()
```

cypher on IRC is reporting that switching to the fork of qt5reactor at https://github.com/gridsync/qt5reactor should resolve the issue.

**Comment by crwood** [comment]
Thanks again for the report!

This seems to be consistent with the upstream issue reported [here](https://github.com/sunu/qt5reactor/issues/10) (which the fork that Windows builds are using is supposed to fix, I believe). In any case, I'll be looking into this further very soon as "Fix Windows builds" is fast-approaching on my TODO list. :)

**Comment by crwood** [comment]
Okay, this was fixed in a905e2b.

Please know though that Windows builds (or any builds, for that matter) are still not quite ready for public consumption and that you're very likely to experience similar packaging-related issues until I finish working out the remaining kinks in the build process.. I'm aiming to have everything smoothed out for a first release by June, however, and would love to hear about any other issues you might encounter before (or even after) then.

In any case, thanks again for pointing this out! Much appreciated!
